tools/config-generator/main.go

```diff
@@ -694,23 +689,13 @@ func setOutput(fileName string) {
 // main is the script entry point.
 func main() {
 	// Parse flags and sanity check them.
-	prowConfigOutput := ""
 	prowJobsConfigOutput := ""
 	testgridConfigOutput := ""
-	var env = flag.String("env", "prow", "The name of the environment, can be prow or prow-staging")
-	var generateProwConfig = flag.Bool("generate-prow-config", true, "Whether to generate the prow configuration file from the template")
-	var generatePluginsConfig = flag.Bool("generate-plugins-config", true, "Whether to generate the plugins configuration file from the template")
 	var generateTestgridConfig = flag.Bool("generate-testgrid-config", true, "Whether to generate the testgrid config from the template file")
 	var generateMaintenanceJobs = flag.Bool("generate-maintenance-jobs", true, "Whether to generate the maintenance periodic jobs (e.g. backup)")
```

Comment:
It looks a lot of flags here are not needed anymore, but it's fine by me to leave them for now.

Reply:
ack